cli/src/main/java/com/liferay/blade/cli/BladeSettings.java

```diff
@@ -59,6 +64,28 @@ public void load() throws IOException {
 		}
 	}
 
+	public void migrateSettingsIfNecessary() throws IOException {
```

Comment:
This should return a new BladeSettings instance with the corrected settings location.